packages/demand-bloc-trip-create/src/index.ts

```diff
@@ -1,2 +1,6 @@
+import * as yup from 'yup'
+
 export { TripCreateForm } from './TripCreateForm'
+export { PassengerDetailsForm, defaultFieldList, defaultValidationSchema } from './PassengerDetailsForm'
+export { yup }
```

Comment:
maybe it's better to call the variable somehow differently ?

Reply:
good point :+1: